src/App.tsx

```diff
@@ -17,40 +36,56 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={query}
+            onChange={(event) => {
+              setQuery(event.target.value);
+            }}
+          />
+          <input
+            className="Animation-Duration-setter"
+            type="number"
+            onChange={(event) => {
+              setAnimationTime(+event.target.value);
+            }}
           />
         </div>
 
-        <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
+        { isEntered === true && (
+          <div className="dropdown-menu" role="menu">
+            <div className="dropdown-content">
+              <div className="dropdown-item transition__li-item">
+                {visibleMovies.map((people) => {
+                  return (
+                    <p
+                      className="has-text-link transition__image"
+                      style={{ animationDuration: `${animationTime}s` }}
```

Comment:
you didn't understand task correctly. it should be debounce between typing in input and filtering movies. remove animation